src/commands.rs

```diff
@@ -125,17 +124,17 @@ enum RusticCmd {
     /// Show the configuration which has been read from the config file(s)
     ShowConfig(ShowConfigCmd),
 
-    /// Update to the latest rustic release
+    /// Update to the latest stable rustic release
     #[cfg_attr(not(feature = "self-update"), clap(hide = true))]
     SelfUpdate(SelfUpdateCmd),
 
     /// Remove unused data or repack repository pack files
     Prune(PruneCmd),
 
-    /// Restore a snapshot/path
+    /// Restore a snapshot or path
```

Comment:
this allows to restore a complete snapshot (`restore <SNAPID>`) or a path within a snapshot (`restore <SNAPID>:<PATH>`). Maybe something like `Restore (a path within) a snapshot`?